src/components/v2/V2Create/tabs/ReviewDeployTab/index.tsx

```diff
@@ -22,9 +22,9 @@ export default function ReviewDeployTab() {
           marginTop: '4rem',
         }}
       >
-        <Space size="large" direction="vertical">
+        <Space size="large" direction="vertical" style={{ width: '100%' }}>
```

Comment:
noob question: Does this change make a difference?

Reply:
Yep my bad should have commented on this line. It relates to the last point in the description